pkg/ui/workspaces/cluster-ui/src/selectors/activeExecutions.selectors.ts

```diff
@@ -46,9 +47,12 @@ export const selectActiveStatements = createSelector(
   (executions: ActiveExecutions) => executions.statements,
 );
 
-export const selectExecutionStatus = () => {
+export const selectExecutionStatus = (execType: ExecutionType) => {
   const execStatuses: string[] = [];
   for (const execStatus in ExecutionStatus) {
+    if (execType == "statement" && execStatus == ExecutionStatus.Idle) {
```

Comment:
Hmm, is this function to populate the execution status types for the filter? If so, I don't think it is necessary to pass this as a prop, rather you can generate the options statically and pass them directly to the Filter component. Could be missing something though.

If you don't feel like doing that in this PR - some nits:
nit: prefer strict equality for non-nullish checks `==` -> `===`
You can also turn this block into a filter if desired.